parser/parser_test.go

```diff
@@ -1223,3 +1223,21 @@ func TestParse(t *testing.T) {
 		})
 	}
 }
+
+func TestExpressionSizeCodePointLimit(t *testing.T) {
+	p, err := NewParser(Macros(AllMacros...), ExpressionSizeCodePointLimit(-2))
+	if err == nil {
+		t.Fatal("expected NewParser to return an error when given an invalid expression size code point limit")
```

Comment:
Can you include the `error` in the test failure?

Reply:
It's nil, as were are testing that an error did occur. Regardless changed the message.